packages/core/strapi/lib/commands/develop.js

```diff
@@ -131,6 +132,8 @@ function watchFileChanges({ dir, strapiInstance, watchIgnoreFiles, polling }) {
       '**/index.html',
       '**/public',
       '**/public/**',
+      strapiInstance.config.get('server.dirs.public'),
```

Comment:
if you have access to the instance why not do strapi.dirs.public ? it should be set based on the conf ?

Reply:
Mmmmh I see.
What is the purpose of having `strapi.dirs` if it already in the config? Just as a shortcut like `strapi.contentTypes`?